src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -510,39 +496,6 @@ public void InstallPackages_Succeeds_SuccessPropertyIsTrue()
             testSubject.InternalState.BindingOperationSucceeded.Should().BeTrue();
         }
 
-        [TestMethod]
-        public void InstallPackages_Succeeds_SuccessPropertyIsFalse()
```

Comment:
The implementation method is now hard-coded to return `true`, so this test is no longer required (and it was confusingly named in any event; it should have been called `InstallPackages_Fails_SuccessPropertyIsFalse`).